packages/opentelemetry-core/src/index.ts

```diff
@@ -112,7 +112,7 @@ export {
 export { merge } from './utils/merge';
 export { TracesSamplerValues } from './utils/sampling';
 export { TimeoutError, callWithTimeout } from './utils/timeout';
-export { isUrlIgnored, urlMatches } from './utils/url';
+export { isUrlAllowed, isUrlIgnored, urlMatches } from './utils/url';
```

Comment:
Hmm, I'd rather have this utility function duplicated across both `@opentelemetry/instrumenation-fetch` and `@opentelemetry/instrumentation-xhr` than it being part of the public API of a stable package. It gives us more flexibility to change it later if it's internal. Anything that is stable, we have to support 1 year after the next major so the timeline for changing/removing it would likely be > 1 year.

I'll ask the SIG what people think about combining `@opentelemetry/instrumenation-fetch` and `@opentelemetry/instrumentation-xhr` as that has recently come up with #4706 as well - if that goes through we may be able to quite easily de-dupe it when we merge the two packages. That's something for a different PR though.